tests/includes/juju.sh

```diff
@@ -79,14 +85,14 @@ bootstrap() {
     if [ -n "${BOOTSTRAP_REUSE}" ]; then
         echo "====> Reusing bootstrapped juju ($(green "${version}"))"
 
-        OUT=$(juju models --format=json 2>/dev/null | jq '.models[] | .["short-name"]' | grep "${model}" || true)
+        OUT=$(juju models -c "${bootstrapped_name}" --format=json 2>/dev/null | jq '.models[] | .["short-name"]' | grep "${model}" || true)
         if [ -n "${OUT}" ]; then
             echo "${model} already exists. Use the following to clean up the environment:"
             echo "    juju destroy-model --force -y ${model}"
```

Comment:
Interestingly `destroy-model` doesn't take a `-c`, which means this message could be wrong from time to time. Probably should change it to include `juju switch ${bootstrapped_name}`